src_view/view.gd

```diff
@@ -140,3 +144,14 @@ func command_move_up(_value) -> void:
 	%ScriptScroll.scroll_vertical -= 10
 func command_move_down(_value) -> void:
 	%ScriptScroll.scroll_vertical += 10
+func command_jump_beginning(_value):
+	%ScriptScroll.scroll_vertical = 0
+func command_jump_end(_value):
+	var track_scroll = %ScriptScroll.scroll_vertical
+	while %ScriptScroll.scroll_vertical == track_scroll:
+		%ScriptScroll.scroll_vertical += 10000
+		track_scroll += 10000
```

Comment:
If the Script would be exactly 10 000 long, this would cause an infinite loop at the moment. But you could make this a lot simpler by taking the %ScriptBox.size.y value instead. You can not get the max size of the vertical scroll, but getting the maximum size of the scriptbox is a possibility and would make this 'command_jump_end' function a lot simpler without the chance of getting stuck in an infinite loop.
```suggestion
	%ScriptScroll.scroll_vertical = %ScriptBox.size.y + 100
```

Reply:
but track_scroll got 10 000 added so they wouldnt equal anymore?